monitor/monitor.go

```diff
@@ -36,6 +36,7 @@ type Monitor struct {
 	sidechainAcct   sdk.AccAddress
 	bonded          bool
 	executeSlash    bool
+	bootstrapped    bool
```

Comment:
Maybe add a comment explaining what this means.

Reply:
done